### PR TITLE
Removed nested card in Status component

### PR DESCRIPTION
**Pull request by cheepo2109**
## Hey, I just made a Pull Request for #830

CleanShot 2020-05-13 at 02 32 47@2x

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Review by Rugvip — COMMENTED** [review]
Thanks! I think something like [this](https://github.com/spotify/backstage/blob/c61400c6e759b669b31e7d633b836edde2f1521b/packages/core/src/layout/Content/Content.tsx#L41) might be better than a separate component.

File: packages/core/src/layout/InfoCard/InfoCard.tsx [hunk 1]
```diff
@@ -142,6 +150,7 @@ const InfoCard: FC<Props> = ({
   actions,
   cardClassName,
   actionsTopRight,
+  noPadding = true,
```
Comment (Rugvip) [inline packages/core/src/layout/InfoCard/InfoCard.tsx:153]: I don't think we want this to be true by default. Better leave this as false and then use it in the Status component story.

**Review by Rugvip — COMMENTED** [review]
Thank you for the update! Minor fix and then we should be good to go

File: packages/core/src/layout/InfoCard/InfoCard.tsx (hunk 1 above)
Comment (cheepo2109) [reply packages/core/src/layout/InfoCard/InfoCard.tsx:153]: Alright!
I just assumed that this is the default design based on https://github.com/spotify/backstage/issues/830

**Comment by cheepo2109** [comment]
Finally passed new tests 🤞

**Review by stefanalund — APPROVED** [review]
Thanks a lot!

**Pull request merged**